docs/v3.x/concepts/draft-and-publish.md

```diff
@@ -35,18 +35,18 @@ Drafts can be modified and saved at will, until they are ready to be published.
 
 ### Publishing a draft
 
-To publish a draft, click on the **Publish** button.
+To publish a draft, click on the **Publish** button in the top right corner of the content edition interface.
 
 :::warning CAUTION
 Before publishing a draft, make sure it doesn't have relations with other non-published content, otherwise some of the content may not be available through the API.
 :::
 
-When a content is not in a draft anymore, but a published content, it is indicated on the right of the interface, below the Information box.
+When a content is not a draft anymore, but a published content, it is indicated on the right of the interface, below the Information box.
```

Comment:
> but a published content

This needs to be changed but I'm not sure of the best way to word it :thinking: maybe "but has been published"